client/extensions/woocommerce/components/list/list-row/index.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
This is not used but would be used instead of payment-method-row.  I can delete but I felt it was needed to complete the set of table components